src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from 'react';
+import './Navbar.scss';
+
+const Navbar = () => {
+
+  const [hamburgerActive, setHamburgerActive] = useState(false)
+  const [sidebarActive, setSidebarActive] = useState(false)
+    
+  const toggleClass = () => {
+    setHamburgerActive(!hamburgerActive)
+    setSidebarActive(!sidebarActive)
```

Comment:
jeśli obie zależą od jednej rzeczy to wrzuć sobie tutaj tylko active jako 1 state